### DISCUSSION TITLE
Wireguard not working on IOS devices on Wifi

### DISCUSSION THREAD

**Discussion by stamandr** [hassio-addons/addon-wireguard#210] [Q&A]
Wireguard is not working on my IOS devices (IPAD,etc) on Wifi. If I switch to data it works and I can access devices on my LAN. Go back to WiFi and I cannot connect to devices on my LAN. On Android I do not have this problem. Any suggestions on how to get this to work on WiFi only on IOS devices ?

**Comment by frenck** [comment]
You created the same issue on Tailscale, sound to me you have completely different issue. This is not related to any of the add-ons.

**Reply by stamandr** [reply]
Yes,  I created the same on Tailscale. Both Wireguard and Tailscale have issues connecting on LAN WiFi on IOS. It works perfectly on Android. I can get Wireguard to work it I add the following to my config. Tailscale does not have this option.

client_allowed_ips:
    - 192.168.0.0/24
